src/main/java/pl/put/poznan/madness/logic/models/SortResult.java

```diff
@@ -0,0 +1,15 @@
+package pl.put.poznan.madness.logic.models;
+
+import pl.put.poznan.madness.logic.SortingAlgorithm;
+
+public class SortResult<T> {
+  public T[] result;
+  public long elapsedMilliseconds;
+  public SortingAlgorithm usedAlgorithm;
+
+  public SortResult(SortingAlgorithm alg, long time, T[] result) {
+    this.usedAlgorithm = alg;
+    this.elapsedMilliseconds = time;
```

Comment:
@mudziok, @tcichy0 to można spokojnie zmienić na nanosekundy bo przy krótkich listach w wyniku jest ciągle 0, ale to już zostawiam wam do decyzji

Reply:
Poprawione w 52002d9